src/main/java/com/google/devtools/build/lib/rules/genrule/GenRuleBaseRule.java

```diff
@@ -161,8 +161,8 @@ public RuleClass build(
             <p>
               Note that <code>outs</code> are <i>not</i> included in this substitution. Output files
               are always generated into a predictable location (available via <code>$(@D)</code>,
-              <code>$@</code>, <code>$(OUTS)</code> or <code>$(location <i>output_name</i>)</code>;
-              see below).
+              <code>$@</code>, <code>$(OUTS)</code> or <code>$(OUT_ROOT_DIR)</code> or
```

Comment:
please also update this to the new variable name

Reply:
fixed.